tests/integration/test_api.py

```diff
@@ -25,15 +25,26 @@
 
 from inspire_disambiguation import conf
 from inspire_disambiguation.api import train_and_save_distance_model, cluster_from_redis
+from inspire_disambiguation.core.data_models.publication import (
+    Publication,
+)
+from inspire_disambiguation.core.data_models.signature import Signature
+from inspire_disambiguation.core.es.readers import get_signatures
 
 
+@patch("inspire_disambiguation.core.ml.models.Pipeline.score")
 @patch("inspire_disambiguation.core.ml.models.Pipeline.fit")
 @patch("inspire_disambiguation.core.ml.sampling.random.choice")
+@patch("inspire_disambiguation.api.math.ceil")
+@patch("inspire_disambiguation.api.train_validation_split")
```

Comment:
I don't think you should be mocking this. `ceil` is a simple deterministic mathematical function, and if you're moking `train_validation_split` you're not really testing anything, are you? as you're hardcoding the seed, the split is deterministic anyway, so I don't think you need to mock anything there.